ext/standard/tests/http/http_response_header_deprecated_bypass.phpt

```diff
@@ -0,0 +1,40 @@
+--TEST--
+$http_reponse_header using suggested cross-compatible version workaround
```

Comment:
I don't understand what this test is actually testing

Reply:
This tests checks that the cross-compatible code provided in the RFC actually works:

```php
$content = [file_get_contents](http://www.php.net/file_get_contents)('http://example.com/');
if ([function_exists](http://www.php.net/function_exists)('http_get_last_response_headers')) {
    $http_response_header = http_get_last_response_headers();
}
// Use $http_response_header as before
```

Reply:
Oof okay now it clicks

Reply:
I can try to make the title more descriptive :)